announcementsAPI.json

```diff
@@ -288,6 +288,58 @@
         }
       }
     },
+    "/announcements/like": {
+      "put": {
+        "tags": [
+          "AnnouncementsEndpoints"
+        ],
+        "summary": "Add announcement to liked announcements for adopter provided in token. Requires adopter role.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/PutAdopterLikeAnnouncementRequest"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "401": {
+            "description": "Unauthorized"
+          }
+        }
+      }
+    },
+    "/announcements/allLiked": {
```

Comment:
Do we really need this endpoint? Wouldn't it be enough to add another type of filter to the endpoint we already have?

Reply:
Me and my team think that it can create too many logic in the /announcements endpoint, because not only adopter can access it. Maybe we should discuss it with other teams and their members